### PR TITLE
clarify rc usage

### PR DESCRIPTION
**Pull request by ericpaulsen**
adding context to release candidates, and when they should be used.

### CODE REVIEW

File: changelog/index.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ icon:
 
 - Release _candidates_ begin appearing on the **second Wednesday** of each month
   (Coder may create multiple release candidates prior to the actual release)
+
+> We do not provide documentation for release candidates, and they should not be
+> used unless you've been asked to use it by Coder.
```
Comment (deansheather) [inline changelog/index.md:19]:
```suggestion
> used unless you've been instructed to use them by Coder.
```

File: changelog/index.md [hunk 2]
```diff
@@ -14,6 +14,10 @@ icon:
 
 - Release _candidates_ begin appearing on the **second Wednesday** of each month
   (Coder may create multiple release candidates prior to the actual release)
+
+> We do not provide documentation for release candidates, and they should not be
```
Comment (deansheather) [inline changelog/index.md:18]:
```suggestion
> We do not provide documentation for release candidates (any version containing `-rc`), and they should not be
```

**Review by deansheather — COMMENTED** [review]
This should also go into the setup guide, not just the changelog page

**Comment by ericpaulsen** [comment]
> This should also go into the setup guide, not just the changelog page

Noted, I'll also include it on the install page.

File: changelog/index.md (hunk 1 above)
Comment (jawnsy) [reply changelog/index.md]: "unless the Coder team instructs you to do so" or something like that?

File: setup/installation.md [hunk 3]
```diff
@@ -75,9 +75,10 @@ kubectl config set-context --current --namespace=coder
 
    _Note: If you omit `--version`, you'll upgrade to the latest version._
 
-   > We do not provide documentation for release candidates (any version containing
-   > `-rc`), and they should not be used unless you've been instructed to use them
-   > by Coder.
+   > We do not provide documentation for release candidates, and you should not
+   use them unless you've been instructed to do so by Coder. You can identify
+   release candidates by the presence of `-rc` in the version number (e.g.,
+   `1.16.0-rc.1`).
```
Comment (deansheather) [inline setup/installation.md:81]: We should clarify that omitting `--version` will not attempt to use a release candidate unless you also specify `--devel`.

File: setup/installation.md [hunk 4]
```diff
@@ -73,8 +73,9 @@ kubectl config set-context --current --namespace=coder
...
    ```
 
-   _Note: If you omit `--version`, you'll upgrade to the latest version._
-
+   > If you omit `--version`, you'll upgrade to the latest version, excluding
+   > release candidates (RCs). To include RCs, provide the `--devel` flag.
+   >
    > We do not provide documentation for release candidates, and you should not
    use them unless you've been instructed to do so by Coder. You can identify
    release candidates by the presence of `-rc` in the version number (e.g.,
```
Comment (deansheather) [inline setup/installation.md]: `release candidate` => `RC`, since you abbreviated it above

**Review by deansheather — APPROVED** [review]

**Review by khorne3 — APPROVED** [review]

**Pull request merged**